examples/ColorSpaceInterpolations.elm

```diff
@@ -0,0 +1,231 @@
+module ColorSpaceInterpolations exposing (main)
+
+{-| This module shows how to build some simple colour space palettes.
+-}
+
+import Browser
+import Color exposing (Color, rgb255)
+import Hex
+import Html exposing (Html, div)
+import Html.Attributes exposing (class, for, href, id, selected, style, type_, value)
+import Html.Events exposing (onInput)
+import Interpolation exposing (Interpolator, hcl, hclLong, hsl, hslLong, lab, rgb)
+import Scale exposing (QuantizeScale)
+
+
+css : String
+css =
+    """
+body {
+    font-family: Sans-Serif;
+}
+
+body, html {
+    width: 100%;
+    position: relative;
+}
+
+.wrapper {
+    margin: 25px;
+    width: 90%;
+    position: relative;
+}
+
+.palette {
+    display: flex;
+    width: 100%;
+    margin: 2px 0 20px;
+}
+
+.palette div {
+    height: 60px;
+    flex: 1;
+}
+
+.controls {
+    margin-bottom : 20px;
+    display: flex;
+}
+
+.controls div {
+    margin : 0 20px 10px;
+}
+
+.controls input,
+.controls select {
+    margin-right : 10px;
+}
+"""
+
+
+type alias Model =
+    { fromColorValue : String
+    , toColorValue : String
+    , count : Int
+    }
+
+
+init : Model
+init =
+    { fromColorValue = "#4682b4"
+    , toColorValue = "#ff0000"
+    , count = 10
+    }
```

Comment:
I think making the start colours such that it clearly demonstrates some of the negative properties of RGB would be nice to "sell" this feature. For example, below you see how sRGB makes a "muddy" region in the middle.

<img width="818" alt="Screenshot 2020-03-09 at 15 05 24" src="https://user-images.githubusercontent.com/69144/76227588-7383aa80-6217-11ea-92dc-c8e182be2c18.png">